src/z3c/relationfield/README.rst

```diff
@@ -19,7 +19,9 @@ Setup
 
 ``z3c.relationfield.Relation`` is a schema field that can be used to
 express relations. Let's define a schema IItem that uses a relation
-field::
+field:
+
+.. code-block:: python
 
```

Comment:
Yes, it's all about the highlighting :)
Also the change from .txt to .rst (IMO restructured text files should always have the extension .rst).

With my vim configuration, native doctest blocks are not highlighted at all. If not ``.. code-block::`` I'd propose to leave the literal blocks ``::``, as they are at least shown visually different from the rest of the text.
Of course, that's also dependent on the editor's settings...